File: ProjectManager/ClientApp/src/app/projects-list/Project.ts
export class Project {
    projectId: number;
    name: string;
    description: string;
    createDateTime?: Date;
    deadLine: Date;
    isActive: boolean;

    constructor(ProjectId: number, Name: string, Description: string, DeadLine: Date, isActive: boolean, CreateDateTime?: Date) {
        this.projectId = ProjectId;
        this.name = Name;
        this.description = Description;
        this.createDateTime = CreateDateTime;
        this.deadLine = DeadLine;
        this.isActive = isActive;
    }

}

File: ProjectManager/Models/Dto/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjectManager.Models.Dto
{
    public class Project
    {
        public int ProjectId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreateDateTime { get; set; }
        public DateTime DeadLine { get; set; }
        public bool isActive { get; set; }
        public virtual ICollection<ProjectStudent> ProjectStudents { get; set; }
        public ICollection<Task> Tasks { get; set; }
    }
}
